src/codesquad/sunny/race/MonsterRace.java

```diff
@@ -0,0 +1,81 @@
+package codesquad.sunny.race;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Random;
+
+public class MonsterRace {
+    private BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+    private int monsterCount;
+    private int attemptCount;
+    private final int count = 5;
```

Comment:
이런 상수의 경우
```suggestion
    private static final int COUNT = 5;
```

와 같은 형태로 정의하는건 어떨까요?

그리고 count가 어떤 count를 의미하는지 알 수 있도록 명확한 의미를 명시해주는게 좋을것 같습니다.

Reply:
이럴때 static을 써야하는군요. static에 대한 연구를 더 해보겠습니다. 아직까지 언제 써야할지 판단이 안되서 그냥 다빼버렸습니다.